wss/listener.go

```diff
@@ -18,106 +18,148 @@ package wss
 
 import (
 	"crypto/tls"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/openziti/identity"
+	transporttls "github.com/openziti/transport/v2/tls"
+
 	"github.com/gorilla/mux"
 	"github.com/gorilla/websocket"
 	"github.com/michaelquigley/pfxlog"
-	"github.com/openziti/identity"
 	"github.com/openziti/transport/v2"
-	"github.com/openziti/transport/v2/ws"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
-	"io"
-	"net/http"
 )
 
-var upgrader = websocket.Upgrader{}
+var (
+	upgrader = websocket.Upgrader{}
+
+	browZerRuntimeSdkSuites = []uint16{
+		//vv WASM-based TLS1.3 suites
+		tls.TLS_AES_256_GCM_SHA384,
+		tls.TLS_CHACHA20_POLY1305_SHA256,
+		tls.TLS_AES_128_GCM_SHA256,
+		//^^
+	}
+)
 
-type wssListener struct {
+type wsListener struct {
```

Comment:
Is there a reason this struct was name changed from `wssListener` to `wsListener` but it is in the `wss` folder?

Reply:
I changed it to wssListener...